cookbook/evals/accuracy/accuracy_with_given_answer.py

```diff
@@ -0,0 +1,20 @@
+from typing import Optional
+
+from agno.agent import Agent
+from agno.eval.accuracy import AccuracyEval, AccuracyResult
+from agno.models.openai import OpenAIChat
+from agno.tools.calculator import CalculatorTools
+
+evaluation = AccuracyEval(
+    agent=Agent(
+        model=OpenAIChat(id="gpt-4o-mini"),
+        tools=[CalculatorTools(add=True, multiply=True, exponentiate=True)],
+    ),
+    prompt="What is 10*5 then to the power of 2? do it step by step",
+    expected_answer="2500",
```

Comment:
Can this then be removed?

Reply:
We always need `prompt` and `expected_answer` on initialization

Then you can run an agent and evaluate the generated answer, or provide an answer and evaluate it directly (`run_with_given_answer`)